src/app/auth/login-page/login-page.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="login">
+  <div class="login-form">
+    <label class="login-form-user-name">
+      User name: <input type="text" [(ngModel)]="userName"/>
+    </label>
+    <label class="login-form-password">
+      Password: <input type="text" [(ngModel)]="password"/>
```

Comment:
It's better to add `type="password"` to password input.

Reply:
Fixed